content/useStyles: a hook to manage your styles - Agustín Prieto - EN.md

```diff
@@ -0,0 +1,65 @@
+![Banner React Native](images/useStylesBanner.jpg)
+
+# useStyles: a hook to manage styles in React Native
+
+¿Does anyone know how to properly manage styles in React Native? I don't. 
+
+I Have been working with this platform for a couple of years now and couldn't get a concise way of using styles. This last year I've tried some libraries with novel concepts as Styled Components, but I couldn't get used to them. As a result of this dissatisfaction with the current alternatives, useStyles was born.
+
+![One hook to rule them all](images/useStylesMeme.jpg)
+
+## What is useStyles and how can it help me?
+
+Organizing styles in React Native can be tricky, a headache if you don't apply good programming practices. useStyles comes with the idea of ​​alleviating this burden by providing the tools so that you can generate applications where the styles have their own place and can reside harmoniously with the components.
+
+If we summarize what useStyles does: it is a hook, which allows you to consume the styles of your application in an efficient, simple, and orderly way. So if it appeals to you the idea of ​​improving the way you write your styles and thus generate cleaner applications, I invite you to visit the [repository](https://github.com/rootstrap/react-native-use-styles ) and try it out for yourself.
+
+Next, we will see an example of use.
+
+## Usage
+ 
+The following example shows how we would apply styles natively to generate a simple container with text inside.
+
+https://gist.github.com/agustito37/cca8292de8f3639972af9d9130dc2784
+
+Seen this way, it doesn't look like bad code, does it? Now, what happens when the component starts to grow.
+
+The component a few weeks later, a few very rough weeks later:
+https://gist.github.com/agustito37/d89db60561fbe5073e340f3b6c6c35b4
+
+As you can see, our clean component began to be slowly eaten by our ugly styles. While there are many ways to improve this code, since React Native does not have a clear and simple way to handle styles, these bad practices are common.
+
+Now let's implement the same but with useStyles:
+https://gist.github.com/agustito37/2f49201ea6f7a7cfce3318181f6e00c4
+
+Excuse me :cry:. My eyes were filled with joy when I saw an organized code. Sniff. Maybe this reminds you a little bit of CSS styles; although useStyles is inspired by this concept; this is a solution created especially for React Native, it has little to do with CSS. 
```

Comment:
```suggestion
Excuse me :cry:. My eyes were filled with joy when I saw an organized code. Sniff. Maybe this reminds you a little bit of CSS styles. Although useStyles is inspired by this concept, this is a solution created especially for React Native; it has little to do with CSS. 
```

I think the sentence is not that clear with those semicolons. WDYT?

Reply:
It could be, I abuse a little of semicolons.